packages/template-retail-react-app/app/pages/checkout/partials/shipping-address.jsx

```diff
@@ -6,7 +6,7 @@
  */
 import React, {useState} from 'react'
 import {nanoid} from 'nanoid'
-import {defineMessage, useIntl} from 'react-intl'
+import {FormattedMessage, defineMessage, useIntl} from 'react-intl'
```

Comment:
`FormattedMessage` doesn't seemed to be used in this file

Reply:
Removed unused import